### DISCUSSION TITLE
403Forbidden

### DISCUSSION THREAD

**Discussion by mailisme** [Rapptz/discord.py#9510] [Q&A] [answered]
Why?
I have given the robot all bot permissions (including applications.commands)
help me pls🙏

![image](https://github.com/Rapptz/discord.py/assets/112573163/dcd6734c-cbfc-4803-b606-fc5569e291f1)

**Comment by Duued** [comment]
What are you specifically trying to do?

**Reply by mailisme** [reply]
make a slash command

**Comment by Duued** [comment, marked as answer]
Nevermind I saw it.. for future reference please do not send screenshots. 

Anyways, since you're trying to guild sync it either means that
- the bot wasn't invited with application_commands - kick the bot and reinvite it
- the bot isn't a member of the target guild 
- you're using the wrong ID

Check all of these and try again!

**Reply by mailisme** [reply]
OK
I`ll try it

**Reply by LostLuma** [reply]
> you're using the wrong ID

This one's the case, there's no guild with ID `893111319140630528`.
